docs/syntax_and_semantics/c_bindings/lib.md

```diff
@@ -17,3 +17,14 @@ Attributes are used to pass flags to the linker to find external libraries:
 * `@[Link(framework: "Cocoa")]` will pass `-framework Cocoa` to the linker (only useful in macOS).
 
 Attributes can be omitted if the library is implicitly linked, as in the case of libc.
+
+## Reflection
+
+Lib functions are visible in the macro language anywhere in the program.
```

Comment:
I don't think it's clear what "anywhere" means here. Also, probably we want to link to the API.

```suggestion
Lib functions are visible in the macro language using the method [`TypeNode#methods`](https://crystal-lang.org/api/1.6.2/Crystal/Macros/TypeNode.html#methods%3AArrayLiteral%28Def%29-instance-method):
```

Reply:
"anywhere" might not be very clear, but it's important. `TypeNode#methods` for other types only works within method context. The point we're mentioning this here is that for lib types it also works outside of methods.

Reply:
but if it's important, it should be clear. anyway, we can leave it for a future improvement.